pytest_unordered.py

```diff
@@ -24,11 +27,14 @@ def __eq__(self, actual: Iterable) -> bool:
         extra_left, extra_right = _compare_eq_unordered(self, actual)
         return not extra_left and not extra_right
 
+    def __ne__(self, actual: Iterable) -> bool:
+        return not self.__eq__(actual)
```

Comment:
Why is this required?

Reply:
If you don't do this, some tests I added fail because unordered "unequality" is not checked properly

I think it is because `UnorderedList` is a subclass of `list`, which has an `__ne__` that does not goes through `__eq__`.

Reply:
Will the tests pass with this implementation?

```suggestion
    def __ne__(self, actual: Iterable) -> bool:
        return not self == actual
```

Reply:
yep that works too, I'll change it to that